Vostok.Logging.Microsoft/Adapters/MicrosoftVostokLogAdapter.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Internal;
+using Vostok.Logging.Abstractions;
+using Vostok.Logging.Abstractions.Wrappers;
+using Vostok.Logging.Formatting;
+using VostokLogLevel = Vostok.Logging.Abstractions.LogLevel;
+using MicrosoftLogLevel = Microsoft.Extensions.Logging.LogLevel;
+
+namespace Vostok.Logging.Microsoft.Adapters
+{
+    [PublicAPI]
+    public sealed class MicrosoftVostokLogAdapter : ILog
+    {
+        private const int NullEventId = 0;
+        private readonly Func<FormattedLogValues, Exception, string> messageFormatter = MessageFormatter;
+
+        private readonly ILogger logger;
+        private readonly OutputTemplate template = OutputTemplate.Parse(
+            $"{{{WellKnownProperties.TraceContext}:w}}" +
+            $"{{{WellKnownProperties.OperationContext}:w}}" +
+            $"{{{WellKnownProperties.SourceContext}:w}}" +
+            $"{{{WellKnownTokens.Message}}}");
+
+        public MicrosoftVostokLogAdapter(ILogger logger)
```

Comment:
1. No need Adapters folder, this whole module about it.
2. Maybe simplify name to VostokLog.
3. Add settings class: `MicrosoftVostokLogAdapter([NonNull] ILogger logger, [CanBeNull] MicrosoftVostokLogAdapter Settings settings)`
https://git.skbkontur.ru/ke/vostok.logging.kontur.legacy/-/blob/master/Vostok.Logging.Kontur.Legacy/KonturLoggingLog.cs#L48